lib/task-graph.js

```diff
@@ -522,6 +524,35 @@ function taskGraphFactory(
         }));
     };
 
+    TaskGraph.prototype.updateGraphProgress = function(progressData) {
+        var pattern = /\d{1,3}\.?\d{0,2}\%/;
```

Comment:
why not restrict the digit number after the decimal point? I think both 12.89% and 12.892% is OK